From 618e6ba587202c5871558d7839252bb9649c83e5 Mon Sep 17 00:00:00 2001
From: Eric Paris <eparis@redhat.com>
Date: Wed, 10 Sep 2014 14:49:26 -0400
Subject: [PATCH 2/3] KUBE_NO_GODEPS to not use in tree godeps

If this is set, and it normally shouldn't be, the config file will not
add the in tree Godeps to the GOPATH.  So all of them must be solved by
the KUBE_EXTRA_GOPATH
---
 hack/config-go.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hack/config-go.sh b/hack/config-go.sh
index 70a4355..7a2fe49 100644
--- a/hack/config-go.sh
+++ b/hack/config-go.sh
@@ -111,7 +111,9 @@ kube::setup_go_environment() {
     GOPATH=${GOPATH}:${KUBE_EXTRA_GOPATH}
   fi
   # Set GOPATH to point to the tree maintained by `godep`.
-  GOPATH="${GOPATH}:${KUBE_REPO_ROOT}/Godeps/_workspace"
+  if [[ -z ${KUBE_NO_GODEPS:-} ]]; then
+    GOPATH="${GOPATH}:${KUBE_REPO_ROOT}/Godeps/_workspace"
+  fi
   export GOPATH
 
   # Unset GOBIN in case it already exists in the current session.
-- 
1.9.3

